contribs/drt/src/main/java/org/matsim/contrib/drt/routing/DrtRoutingModule.java

```diff
@@ -61,15 +66,22 @@
 	private final PopulationFactory populationFactory;
 	private final RoutingModule walkRouter;
 	private final DrtStageActivityType drtStageActivityType;
+	private final PlansCalcRouteConfigGroup plansCalcRouteConfig;
+
+	public DrtRoutingModule( DrtConfigGroup drtCfg, @Named(DvrpRoutingNetworkProvider.DVRP_ROUTING) Network network,
+					 @Named(DvrpTravelTimeModule.DVRP_ESTIMATED) TravelTime travelTime,
+					 TravelDisutilityFactory travelDisutilityFactory,
+					 @Named(TransportMode.walk) RoutingModule walkRouter, Scenario scenario ) {
+		// constructor was public when I found this, and has to remain public.  --> Now passing scenario as an argument so that
+		// we have at least some flexibility without having to change the argument list.  kai, jul'19
 
-	public DrtRoutingModule(DrtConfigGroup drtCfg, @Named(DvrpRoutingNetworkProvider.DVRP_ROUTING) Network network,
-			@Named(DvrpTravelTimeModule.DVRP_ESTIMATED) TravelTime travelTime,
-			TravelDisutilityFactory travelDisutilityFactory, PopulationFactory populationFactory,
-			@Named(TransportMode.walk) RoutingModule walkRouter) {
-		this.drtCfg = drtCfg;
+		LOGGER.setLevel( Level.DEBUG );
+
+		this.drtCfg = drtCfg ; // can't take out of normal config; don't know why.  kai, jul'19
```

Comment:
There may be multiple drt config groups. So it needs to be explicitly passed via constructor.